sdks/python/apache_beam/ml/transforms/embeddings/huggingface_test.py

```diff
@@ -44,6 +45,7 @@
 except ImportError:
   tft = None
 
+_HF_TOKEN = os.environ.get('HF_INFERENCE_TOKEN')
```

Comment:
Can we add this to our actions yaml file so that we verify this runs?

Reply:
I have added this to the python precommit Coverage actions file in the PR.